TestingExplainer/ViewController.swift

```diff
@@ -127,6 +127,7 @@ var overlayWindowAssociatedObjectKey: UInt8 = 0
 
 extension UIViewController {
     
+    // Yay stored properties in extensions
```

Comment:
Not sure if sarcasm, I'm a bit torn on this approach: hiding storage and using objc runtime, but means users of this functionality just have one step